City-of-Helsinki/drupal-helfi-etusivu
# UHF-5461 news rss feed

# News rss feed [UHF-5459](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5461 )
Added rss feed to news archive view

## What was done
* Added rss feed to news archive view

## How to install
* Make new

## How to test
* You need some news item -content. 2-3 should be enough for testing (mind the language)
   * Make sure at least one news item has "tag" taxonomy term reference for testing the filtering
   * "Lead" -field is used as feed's description
   * Check the id's of those taxonomy terms in order to use them as filters
* go to /news/rss
   * You should see all news content items
* Add query parameter `?tags[]=123` (123 being the id of the tag)
   * You should see filtered list of news.
